### DISCUSSION TITLE
No map on STYTJ02YM (Mi Robot Vacuum-Mop P)

### DISCUSSION THREAD

**Discussion by 0k00l** [rand256/valetudo#528] [General]
Hi, i don't have map displayed on my valetudo-rooted vaccum. Space mapping works fine because robots works as expected but no map on "Map Tab". Could you tell me what i should do?
Some info about robot:
![obraz](https://github.com/user-attachments/assets/43012d55-b721-4f96-951b-eef7b0b48215)

Map tab:
![obraz](https://github.com/user-attachments/assets/5ea4aaee-cda3-4cbc-b680-58a96b02687f)

**Comment by pidator** [comment]
You're asking in the wrong repo, check [this one](https://github.com/Hypfer/Valetudo).

**Comment by 0k00l** [comment]
Sorry and thx :)
